core/io/file_access_zip.cpp

```diff
@@ -200,7 +200,7 @@ bool ZipArchive::try_open_pack(const String &p_path, bool p_replace_files) {
 
 		unz_file_info64 file_info;
 		err = unzGetCurrentFileInfo64(zfile, &file_info, filename_inzip, sizeof(filename_inzip), NULL, 0, NULL, 0);
-		ERR_CONTINUE(err != UNZ_OK);
+		ERR_CONTINUE(err != UNZ_EOF && err != UNZ_OK);
```

Comment:
Was this prompted by a warning? It may change the function logic.

Reply:
I was looking only at one place with macros, but minizip seems to use also macros from zlib, so I just removed this, because it would actually change the logic of program.